t/proto_template/scripts/generate.sh

```diff
@@ -19,17 +20,14 @@ function pre_gen_go {
 function post_gen_go {
   echo "running post_gen_go"
   cd $1
-  go mod tidy
+  go mod tidy  
   cd -
 }
 
 function pre_gen_node {
   echo "running pre_gen_node"
 
-  proto_version_key='proto_version:'
-  proto_version_file='version.yaml'
-
-  PROTO_VERSION=${PROTO_VERSION:-$(grep $proto_version_key $proto_version_file | cut -c $((${#proto_version_key} + 1))- | xargs)}  
+  PROTO_VERSION=${PROTO_VERSION:-$(grep 'proto_version:' 'version.yaml' | cut -d':' -f2 | xargs echo)}
```

Comment:
this will use the env variable if its set or fallback to the one in the version file